src/game/overlay.c

```diff
@@ -17,8 +17,12 @@
 #define MAX_PICKUP_COLUMNS 4
 #define MAX_PICKUPS 16
 
-static int16_t BarOffsetY[6] = { 0 };
-static DISPLAYPU Pickups[MAX_PICKUPS] = { 0 };
+static struct {
+    TEXTSTRING *ammo_text;
+    TEXTSTRING *fps_text;
+    int16_t bar_offset_y[6];
+    DISPLAYPU pickups[MAX_PICKUPS];
+} S = { 0 };
```

Comment:
are you not going to convert the S -> m_ as well?

Reply:
Good point, fixed